validate.py

```diff
@@ -1,111 +1,125 @@
-# validate.py - Unizin UDW validator
+# validate.py - Unizin Data Warehouse validator
 #
-# Copyright (C) 2019 University of Michigan Teaching and Learning
+# Copyright (C) 2019 University of Michigan ITS Teaching and Learning
 
-UNIZIN_FILE_FORMAT = "unizin_{table}.csv"
+# Local modules
+from dbqueries import QUERIES
 
-## don't modify anything below this line (except for experimenting)
-
-import sys, os, csv, itertools, argparse, smtplib, tempfile, json
+# Standard modules
+import json, logging, os, smtplib, sys
+from datetime import datetime
 
+# Third-party modules
 from email.mime.text import MIMEText
-from collections import OrderedDict 
-
-from dateutil import parser as dparser
-
-import psycopg2
-
-import numpy as np
 import pandas as pd
+import psycopg2, pytz
 
-import dbqueries
-
-from collections import OrderedDict
-from operator import itemgetter
-
-from tqdm import tqdm
+# Global variables
 
+logging.basicConfig(level="DEBUG")
+logger = logging.getLogger(__name__)
 
 try:
-    with open(os.getenv("ENV_FILE", "/unizin-csv-validation/config/env.json")) as f:
+    with open(os.getenv("ENV_FILE", "config/env.json")) as f:
         ENV = json.load(f)
 except FileNotFoundError as fnfe:
-    print("Default config file or one defined in environment variable ENV_FILE not found. This is normal for the build, should define for operation")
+    logger.debug("Default config file or one defined in environment variable ENV_FILE not found. This is normal for the build, should define for operation.")
```

Comment:
Minor but this should probably be at least an .info. 

The level is debug by default though but it should be a more noticeable error.

Reply:
Done. See latest [commit](https://github.com/tl-its-umich-edu/unizin-validation/pull/7/commits/f7b1096af1e237e424d0306fb54c6503794c0fca).